dspace-xmlui/src/main/java/org/dspace/app/xmlui/aspect/artifactbrowser/SendItemRequestAction.java

```diff
@@ -105,7 +107,10 @@ public Map act(Redirector redirector, SourceResolver resolver, Map objectModel,
         }
         
         Item item = (Item) dso;
-        String title = "";
+        String title = itemService.getMetadataFirstValue(item, "dc", "title", null, Item.ANY);
+        
+        title = title != null ? title : I18nUtil
+                            .getMessage("jsp.general.untitled", context);
```

Comment:
Would it not be better to use item.getName() (https://github.com/DSpace/DSpace/blob/master/dspace-api/src/main/java/org/dspace/content/Item.java#L334) & then use StringUtils.isNotBlank() instead of a null check (this will ensure a blank title ("") is also sent as untitled).